### PR TITLE
Migrate chart_trace.dart to null safety.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/devtools_app/lib/src/charts/chart_trace.dart [hunk 1]
```diff
@@ -198,11 +192,11 @@ class Trace {
   ///            | * |
   ///            !___!
   ///
-  Path _createSquare() {
+  Path? _createSquare() {
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/charts/chart_trace.dart:195]: this path will be non-null

File: packages/devtools_app/lib/src/charts/chart_trace.dart [hunk 2]
```diff
@@ -223,23 +217,23 @@ class Trace {
   ///         / * \
   ///       ./_____\.
   ///
-  Path _createTriangle() {
+  Path? _createTriangle() {
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/charts/chart_trace.dart:220]: non-null

File: packages/devtools_app/lib/src/charts/chart_trace.dart [hunk 3]
```diff
@@ -251,21 +245,21 @@ class Trace {
   //             \ * /
   //              \ /
   //               `
-  Path _createTriangleDown() {
+  Path? _createTriangleDown() {
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/charts/chart_trace.dart:248]: non-null

File: packages/devtools_app/lib/src/charts/chart_trace.dart (hunk 2 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/charts/chart_trace.dart:220]: done

File: packages/devtools_app/lib/src/charts/chart_trace.dart (hunk 3 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/charts/chart_trace.dart:248]: done

File: packages/devtools_app/lib/src/charts/chart_trace.dart (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/charts/chart_trace.dart]: done

**Review by kenzieschmoll — APPROVED** [review]

**Comment by polina-c** [comment]
tested

**Pull request merged**